### PR TITLE
feat: add marital status and religion

### LINKED ISSUE #2475: Add new fields to Social History section in Patient Info
## User Story

As a user, I want to see Marital Status and Religious Affiliation in the social history section, so that I can better understand the patient.

## Acceptance Criteria

- [ ] Add new field "Marital Status" to social history
- [ ] Add new field "Religious Affiliation" to social history
- [ ] If the data is missing the fields are seen below in the Unavailable Info section
- [ ] Update tests

## Scenarios

- I open up an eCR in the viewer
- I look at the Social History section of an eCR
- I can now see Marital Status and Religious Affiliation sections

## Design Notes

_Owned by design team - Attach link to design in dev mode and/or prototype, as well as any notes on visual details, interaction expectations, accessibility considerations, copy requirements_

[Link to Figma design in Dev mode](https://www.figma.com/design/zAPyOEYwNtx3f2Y47aoq0G/eCR-Viewer-(Aug-Dec-2024)?node-id=7006-69928&node-type=canvas&t=LtmVvUYPEAxYG2jI-0)

- 

_Screenshots below should not be used for working on this ticket - always refer to Figma design for most up-to-date designs:_

### PR DESCRIPTION
**Pull request by mcmcgrath13**
# PULL REQUEST

## Summary
Add Marital Status and Religious Affiliation to Social History. Corresponds to https://github.com/skylight-hq/FHIR-Converter/pull/17.

Along the way, cleaned up fhirpaths to use the magic of `value` instead of `iif(value....)` and made sure `evaluateValue` had coverage of the things there.

## Related Issue
Fixes #2475

## Acceptance Criteria
- [x] Add new field "Marital Status" to social history
- [x] Add new field "Religious Affiliation" to social history
- [x] If the data is missing the fields are seen below in the Unavailable Info section
- [x] Update tests

## Additional Information
image

### CODE REVIEW

File: containers/fhir-converter/Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 FROM mcr.microsoft.com/dotnet/sdk:6.0 AS build
 
 # Download FHIR-Converter
-RUN git clone https://github.com/skylight-hq/FHIR-Converter.git --branch v7.0-skylight-13 --single-branch /build/FHIR-Converter
+RUN git clone https://github.com/skylight-hq/FHIR-Converter.git --branch mcm/add-soc-hist --single-branch /build/FHIR-Converter
```
Comment (mcmcgrath13) [inline containers/fhir-converter/Dockerfile:4]: TODO: update this to released version before merging

File: containers/fhir-converter/tests/integration/test_FHIR-Converter.py [hunk 2]
```diff
@@ -92,7 +92,7 @@ def test_ecr_conversion_with_rr(dir_name, snapshot):
         "root_template": "EICR",
         "rr_data": rr_data,
     }
-    ecr_conversion_response = httpx.post(CONVERT_TO_FHIR, json=request)
+    ecr_conversion_response = httpx.post(CONVERT_TO_FHIR, json=request, timeout=20.0)
```
Comment (austin-hall-skylight) [inline containers/fhir-converter/tests/integration/test_FHIR-Converter.py:95]: Do we need to keep this or were you just troubleshooting some stuff while running locally?

File: containers/ecr-viewer/src/app/tests/utils.test.tsx [hunk 3]
```diff
@@ -64,6 +64,28 @@ describe("Utils", () => {
 
       expect(actual.availableData[0].value).toEqual("Other");
     });
+    it("should return relgion if available", () => {
```
Comment (austin-hall-skylight) [inline containers/ecr-viewer/src/app/tests/utils.test.tsx:67]: 🔬I wouldn't have even bothered if this was the only thing needing changed but you may as well fix it when you're reverting the Dockerfile changes 
```suggestion
    it("should return religion if available", () => {
```

File: containers/fhir-converter/tests/integration/test_FHIR-Converter.py (hunk 2 above)
Comment (mcmcgrath13) [reply containers/fhir-converter/tests/integration/test_FHIR-Converter.py:95]: I've seen these timeouts occasionally locally, so I figured committing the bump would generally be useful (as I know you also ran into the same on occasion)

**Review by austin-hall-skylight — APPROVED** [review]

**Pull request merged**